src/Microsoft.VisualStudio.Services.Agent/Constants.cs

```diff
@@ -308,6 +308,7 @@ public static class Release
                 // Keep alphabetical
                 //
                 public static readonly string AgentReleaseDirectory = "agent.releaseDirectory";
+                public static readonly string AgentReleaseRootDirectory = "agent.releaseRootDirectory";
```

Comment:
Around 2.115 agent.tempdirectory was added. The agent handles cleaning it up (best effort, before/after each task runs iirc). Is that the type of folder you're looking for?